File: FITCC.UI/admin_fitcc/lib/models/dogadjaj.dart
import 'dart:convert';
class Dogadjaj {
  late int dogadjajId;
  late String naziv;
late int? trajanje;
late DateTime? pocetak;
late DateTime? kraj;
  late String napomena; 
  late int agendaId;
  late String lokacija;

  Dogadjaj();

  Dogadjaj.full(this.dogadjajId, this.naziv, this.trajanje, this.pocetak, this.kraj,
      this.napomena, this.agendaId, this.lokacija);

  factory Dogadjaj.fromJson(Map<String, dynamic> map) {
    return Dogadjaj.full(
        map['dogadjajId'],
        map['naziv'],
        map['trajanje'],
        DateTime.tryParse(map['pocetak']),
        DateTime.tryParse(map['kraj']),
        map['napomena'],
        map['agendaId'],
        map['lokacija']);
  }

  static Dogadjaj DogadjajFromJson(String json) {
    final data = const JsonDecoder().convert(json);
    return Dogadjaj.fromJson(data);
  }
}

File: FITCCRS2.Model/Dogadjaj.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace FITCCRS2.Model
{
    public class Dogadjaj
    {
        public int DogadjajId { get; set; }
        public string Naziv { get; set; }
        public int? Trajanje { get; set; }
        public DateTime? Pocetak { get; set; }
        public DateTime? Kraj { get; set; }
        public string Napomena { get; set; }
        public int? AgendaId { get; set; }
        public string Lokacija { get; set; }
    }
}
